backends/tc/runtime/send_packet

```diff
@@ -1,15 +1,7 @@
 #!/usr/bin/env python3
```

Comment:
Is this still needed?

Reply:
@vbnogueira Can we replace this code with the other utilities we are using?

Reply:
@fruffy It is currently used by the p4tc tests.

I have a way to update ptf and bf-pktpy with a few extra lines of code, such that we can relicense this send_packet file as Apache-2.0.  I have those changes in this PR, but there are a couple of other PRs that must be approved and merged first before it can be merged: https://github.com/p4lang/p4c/pull/5145